driver.go

```diff
@@ -293,6 +362,31 @@ func (s *Session) existsDB(tx neo4j.Transaction, node graph.Node, config Config)
 	return true, nd.Id, nil
 }
 
+func (s *Session) entityDBIds(tx neo4j.Transaction, ids []string, config Config) ([]int64, []string, error) {
+	var entityDBIds []int64 = make([]int64, 0, len(ids))
+	var entityIds []string = make([]string, 0, len(ids))
+	if len(ids) == 0 {
+		return entityDBIds, entityIds, nil
+	}
+	idrec, err := tx.Run("MATCH (n) WHERE  n.`ls:entityId` OR n.`https://lschema.org/entityId` IN $id return ID(n), n.`https://lschema.org/entityId`, n.`ls:entityId`",
```

Comment:
The WHERE condition is wrong. It should use `WHERE n.propname in $id return ID(n), n.propName`

where `propName` is config.Map(ls.EntityIdTerm)